packages/loader/container-loader/src/container.ts

```diff
@@ -596,6 +596,32 @@ export class Container
 		return this.loader.services.codeLoader;
 	}
 
+	/**
+	 * {@inheritDoc @fluidframework/container-definitions#IContainer.entryPoint}
+	 */
+	public async getEntryPoint?(): Promise<FluidObject | undefined> {
+		if (this._lifecycleState === "closed" || this._lifecycleState === "disposed") {
```

Comment:
i haven't been keeping close track of the difference between these two. I think we might want this accessible when closed, but not sure

Reply:
Sounds right based on what I understand about it, true. While closed it should still be possible to read from the containter. Updating it to only check for disposed, and reject the new promise if "disposed" fires before "contextChanged". It does leave me thinking what (if anything) we should do if the "closed" event fires while we're waiting here, or if it should be possible in the first place.